unlock-app/src/components/interface/members/airdrop/AirdropElements.tsx

```diff
@@ -26,13 +26,15 @@ interface AirdropListItemProps {
 }
 
 export function AirdropListItem({
-  value: { recipient, count },
+  value: { recipient, count, email, ...rest },
   onRemove,
 }: AirdropListItemProps) {
   return (
     <div className="flex items-center justify-between w-full px-2 py-1 text-sm bg-white rounded-lg shadow">
       <div className="space-x-2">
-        <span>{addressMinify(recipient)}</span>
+        <span>
+          {addressMinify(recipient)} {email ? `(${email})` : ''}{' '}
```

Comment:
Is there enough space here? 🤔 

We should probably separate it more visually.

Reply:
It looks good enough for now.